e2e/libs/backup/rest.py

```
@@ -53,8 +53,11 @@ def get(self, backup_id, volume_name):
 
     def get_from_list(self, backup_list, backup_id):
         for backup in backup_list["items"]:
-            if backup['metadata']['annotations']['test.longhorn.io/backup-id'] == backup_id:
-                return backup
+            try:
+                if backup['metadata']['annotations']['test.longhorn.io/backup-id'] == backup_id:
+                    return backup
+            except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)